### DISCUSSION TITLE
Search results should be filtered when profile is marked as Private

### DISCUSSION THREAD

**Discussion by ts-0** [community/community#16191] [Other Feature Feedback, Questions, & Ideas]
With the current privacy options, you are still able to find anyone's public activity (e.g. opened issues and comments) using the `involves:username` query on GitHub.

This seems like an oversight, I would expect site-wide search to not return any results when you mark your profile as private.

**Comment by tw4** [comment]
also it would be nice if their social media accounts and company were hidden
